app/assets/stylesheets/application.tailwind.css

```diff
@@ -478,3 +478,25 @@ div.field_with_errors > :is(select) {
 .table-container .empty_state_message {
   display: none;
 }
+
+.samples-app {
```

Comment:
Shouldn't this be something more like `scrollable-wrapper` or something?  We might want to use this on a non-samples table.

Reply:
This actually non-scrollable or at least intended to be non-scrollable. E.G. wrapper that takes up remaining browser size. Not sure what that should be named?